### ISSUE TITLE
Added Tags to XPlat Release Notes

### ISSUE THREAD

**Issue by rfennell** [rfennell/AzurePipelines#426]
### Azure DevOps Extension you are using
Change for XPlat Release Notes for ALM Rangers

Add a tag condition to the @@WILOOP@@, for example in the template below we extract description for work item(s) tagged with RELEASENOTE and then BUSINESSVALUE, followed by a list of all the work items. 

Something like 

```
# Release notes
## Notes for release  ${releaseDetails.releaseDefinition.name}
**Release Number**  : ${releaseDetails.name}
**Release completed** : ${releaseDetails.modifiedOn}
**Compared Release Number**  : ${compareReleaseDetails.name}
 
### RELEASE NOTES
@@WILOOP-RELEASENOTE@@
${widetail.fields['System.Description']}
@@WILOOP-RELEASENOTE@@
 
### BUSINES VALUE
@@WILOOP-BUSINESSVALUE@@
${widetail.fields['System.Description']}
@@WILOOP-BUSINESSVALUE@@
 
---
 
### Associated work items
@@WILOOP@@
* ** ${widetail.fields['System.WorkItemType']} ${widetail.id} ** Assigned by: ${widetail.fields['System.AssignedTo']}  ${widetail.fields['System.Title']}
@@WILOOP@@
 
### Associated commits
@@CSLOOP@@
* **ID ${csdetail.id} ** ${csdetail.message}
@@CSLOOP@@

```

**Comment by wpschaub** [comment]
Thanks for adding this issue @rfennell. This will allow us to generate release notes that are valuable to business and engineering. Let me know when and how we can help.

**Comment by rfennell** [comment]
@wpschaub Is this format OK for your needs

```
@@WILOOP:RELEASENOTE:BUSINESS VALUE@@
${widetail.fields['System.Description']}
@@WILOOP:RELEASENOTE:BUSINESS VALUE@@

```
It allows

- Multiple tags
- They can have spaces in them
- I would use them in an AND when selecting WI

Do you think : (colon) is a reasonable delimiter, I am erring towards making it configurable as a parameter, someone is bound to have used : in a tag

Thoughts?

**Comment by ChrisLGardner** [comment]
Perhaps have a toggle to make it swap to OR instead of AND? Or go for a regex style approach with ; being for AND then | for OR? 

That would let you end up with statements like:

`@@WILOOP(RELEASENOTE;(BUSINESS VALUE|IMPORTANT NOTE))@@`

**Comment by rfennell** [comment]
@ChrisLGardner AND/OR I think is for V-future, lets get this working first

**Comment by rfennell** [comment]
Shipped in 2.7.10
